designsafe/apps/workspace/api/urls.py

```diff
@@ -19,4 +19,5 @@
     re_path(r"^jobs/(?P<operation>\w+)/?$", views.JobsView.as_view()),
     path("jobs", views.JobsView.as_view()),
     path("allocations", views.AllocationsView.as_view()),
+    
```

Comment:
Please remove the whitespace added to this file. In general, it's good practice to look over the changes in each PR and make sure that you're changing the minimum possible amount of code to accomplish the task.